src/main/scala/nl.knaw.dans.easy.deposit/CommandLineOptions.scala

```diff
@@ -40,12 +45,32 @@ class CommandLineOptions(args: Array[String], configuration: Configuration) exte
        |Options:
        |""".stripMargin)
 
-  val runService = new Subcommand("run-service") {
+  private implicit val stateLabelParser: ValueConverter[State] = singleArgConverter[State](State.withName)
+  private implicit val uuidParser: ValueConverter[UUID] = singleArgConverter[UUID](UUID.fromString)
+
+  val runService: Subcommand = new Subcommand("run-service") {
     descr(
       "Starts EASY Deposit Api as a daemon that services HTTP requests")
     footer(SUBCOMMAND_SEPARATOR)
   }
+  val changeState = new Subcommand("change-state") {
+    descr(
+      "Changes the state of a deposit, when changing to SUBMITTED just the state is changed, the rest of the submit-cycle is not started")
+    val doUpdate: ScallopOption[Boolean] = opt(name = "doUpdate", noshort = true, required = false,
+      descr = s"without this argument only the current status is shown in json format")
```

Comment:
This is not what the omission of `--doUpdate` should do: it should run the same code as when `--doUpdate` is provided, but replace side effects (in this case 'saving the new state') with a printed message stating the side effect that would have taken place if `--doUpdate` would have been provided.